### DISCUSSION TITLE
Jarvis-Konfiguration auf einem Pad (iPad)

### DISCUSSION THREAD

**Discussion by treblada** [Zefau/ioBroker.jarvis#2573] [Q&A]
Hallo,
ist es möglich die Konfigurationsseiten von Jarvis auf einem Pad zu benutzen? Ich kann zwar Jarvis grundsätzlich auf dem Pad benutzen, aber es fehlt das Einstellungen-Icon.

**Comment by mcuiobroker** [comment]
Browser öffnen z.B.
`http://192.168.178.154:8082/jarvis/index.html#/configuration/layout`
